src/hooks/persisted/useNovel.ts

```diff
@@ -301,7 +302,9 @@ export const useNovel = (url: string, pluginId: string) => {
             };
           }),
         );
-        showToast(`Deleted ${_chapter.name}`);
+        showToast(
```

Comment:
These can be used in novel, download screen, not only update screen

Reply:
Should I use `common.deletedChapter`?

Reply:
I think `common.deleted` is gud.
eg: deleted chapter abc, deleted 12 chapter, deleted novel xyz, ...